src/Dev/Tasks/FileMigrationHelper.php

```diff
@@ -257,8 +257,14 @@ protected function migrateFile($base, File $file, $legacyFilename)
             return false;
         }
 
-        // Fix file classname if it has a classname that's incompatible with its extention
+        // Ignore extensionless files
         $extension = $file->getExtension();
+        if ($extension === "") {
```

Comment:
Possibly use apostrophes instead of quotes for consistency?

Reply:
I only used doubles as the phpdoc for `File::get_file_extension()` shows:

```
 * Examples:
 * - "myfile" returns ""
```

So not too bothered, I guess we aren't interpolating a string so doubles aren't required